draft-ietf-ppm-dap.md

```diff
@@ -525,21 +529,21 @@ Client could not report 10^6s or -20s.
 
 # Message Transport {#message-transport}
 
-Communications between DAP participants are carried over HTTPS {{!RFC9110}}.
-HTTPS provides server authentication and confidentiality. Use of HTTPS is
-REQUIRED.
+Communications between DAP participants are carried over HTTP {{!RFC9110}}. Use
+of HTTPS {{!RFC8446}} is REQUIRED to provide server authentication and
```

Comment:
Is this the correct reference? RFC8446 defines TLS, not HTTPS.

Reply:
Hmm, good point. RFC 9110 obsoletes RFC 2818 "HTTP Over TLS", so maybe the reference to 9110 suffices?